docs/user.markdown

```diff
@@ -62,6 +63,8 @@ For a quick start introduction, please look [here](./quick.html).
                             recipes set ([user/repo@]branch)
       --dry-run, -n         Prints what would happen, without actually doing the
                             build.
+      --offline             Offline mode. Does not attempt to connect to the
+                            Internet to fetch repositories.
```

Comment:
No `.` at the end of the sentence (incidentally, it has to be changed on `--dry-run` too)